schema_source/aws_serverless_httpapi.py

```diff
@@ -86,6 +86,8 @@ class Route53(BaseModel):
     HostedZoneId: Optional[PassThroughProp] = route53("HostedZoneId")
     HostedZoneName: Optional[PassThroughProp] = route53("HostedZoneName")
     IpV6: Optional[bool] = route53("IpV6")
+    SetIdentifier: Optional[str]  # TODO: add docs
+    Region: Optional[str]  # TODO: add docs
```

Comment:
These are both pass-throughs (i.e. the value, which could be any arbitrary intrinsic function, is passed as-is to the underlying CloudFormation which we don't control):

```suggestion
    SetIdentifier: Optional[PassThroughProp]  # TODO: add docs
    Region: Optional[PassThroughProp]  # TODO: add docs
```